keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -986,19 +1250,17 @@ This section must be completed when targeting alpha to a release.
 
 ###### How can this feature be enabled / disabled in a live cluster?
 
-<!--
-Pick one of these and delete the rest.
--->
+Kubernetes cluster admins can deploy COSI via this high-level flow:
+
+1. Create CRDs and Deployment for COSI controller including any supporting resources like ServiceAccounts
```

Comment:
nit: would you actually want this deployed as a daemon set?

Reply:
In short, no. 

The COSI controller does nothing on a per-node basis. This goes for the COSI sidecar as well. Nothing in COSI mirrors the CSI node plugins because object storage needs no per-host mount. Object storage will always be accessed via the relevant client API from inside a Pod.

There are some projects (I know of one for S3) that mount object storage into a filesystem mount. COSI's stance is that those systems would need a CSI driver instead because they provide filesystem access, even while the backend is object.

Reply:
Thank you for answering.